src/app/treasure-hunt/find-treasure/find-treasure.component.html

```diff
@@ -343,6 +337,24 @@ <h1 class="p-2 lead text-center mb-0">
         </p>
       </div>
     </div>
+
+
+    <div
+      *ngIf="(displayCalculatorType == 'All' || (displayCalculatorType == 'Water') && treasureHunt.currentEnergyUsage.waterUsed == true)"
+      class="card calc-card-item m-2" (click)="openCalculator('cooling-tower-makeup')">
+      <div class="card-body d-flex flex-column p-2 align-items-center">
+        <div class="calc-icon d-flex justify-content-center align-items-center">
```

Comment:
in the CSS file for this component create a new class called "pc-calc-icon" for the background styling, and integrate the same way as the other custom CSS classes 
![image](https://user-images.githubusercontent.com/84926132/186206910-d1f78836-da8d-43c3-95e9-ad55a87e1eb0.png)